codes/classical/Cycle Codes.yml

```diff
@@ -0,0 +1,34 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+
+
+# code id, physical, logical are all lower case
+code_id: cycle
+physical: bits
+logical: bits
+
+
+name: 'cycle code of graphs '
+introduced: '\cite{https://ieeexplore.ieee.org/document/394711}'
+
+
+description: 'Let \(G\) be a finite, undirected, connected graph without loops and multiple edges.The graph \(G\) has (\r\) vertices and \(n\) edges. Its parity check matrix (\H\) is defined as follows.(\H\) is an (\ r \times n\) matrix whose rows are indexed by the vertices of \(G\) and whose columns are indexed by the edges of \(G\), and such that \(H_{ij}=1\) if vertex \(i\) belongs to the edge \(j\) and \(H_{ij}=0\) otherwise.'
+
+
+  rate: 'k=n-r+1.'
+
+notes:
+  - 'Links to more papers \cite{https://doi.org/10.1007/978-3-642-01877-0_21}.'
+
+
+relations:
+
+  cousins:
+    - code_id: surface
```

Comment:
There's not enough information here as to why these codes are cousins, ie., why are they related to this code.

Reply:
To add details, one may mention general chain (or simplicial) complexes.

Reply:
The ref I pointed to re these entries is 10.1007/978-3-642-01877-0; there should be a blurb in there about connections to surface codes. On that note, could consider adding code_id:css and code_id:homological_classical as cousins since there are chain-complex based constructions there (please read those to see if/how they are related).